fixtures/comments.env

```
@@ -1,4 +1,6 @@
 # Full line comment
+qux=thud # fred # other
+thud=fred#qux # other
```

Because a later refactoring could break expected behavior

I would like you to add this, and test it


```suggestion
thud=fred#qux # other
quz=foo#baz # other # more
```